vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/BaseAndroidDevice.kt

```diff
@@ -100,17 +101,19 @@ abstract class BaseAndroidDevice(
         if (androidConfiguration.allureConfiguration.enabled) {
             androidConfiguration.fileSyncConfiguration.pull.add(
                 FileSyncEntry(
-                    androidConfiguration.allureConfiguration.relativeResultsDirectory,
-                    AggregationMode.TEST_RUN
+                    relativePath = androidConfiguration.allureConfiguration.relativeResultsDirectory,
+                    pathRoot = PathRoot.APP_DATA,
```

Comment:
Should be changed back to external storage if allure-kotlin doesn't accept the PR